### ISSUE TITLE
h render method can not use ElButton component

### ISSUE THREAD

**Issue by npm-ued** [element-plus/element-plus#1374]
as the title 
```
import { ElButton } from 'element-plus';
...
render() {
  return h(ELButton,{}, 'test');
}
```
expect:
button with test text
actual：
Uncaught (in promise) ReferenceError: ElButton is not defined

**Comment by BeADre** [comment]
Can you provide a reproduction demo？

**Comment by npm-ued** [comment]
myTestPage
```
<script lang="ts">
import { defineComponent, h } from 'vue';
import TestChild from './testChild.vue';
import { ElButton } from 'element-plus';
export default defineComponent({
  name: 'testPage',
  render() {
    // return h(TestChild, {});
    return h(ElButton, {}, 'test btn');
  }
});
</script>
```
myTestComponent
```
<template>
  <button @click="testClick">{{ test ? 'open' : 'close' }}</button>
</template>
<script lang="ts">
import { defineComponent, ref } from 'vue';
export default defineComponent({
  name: 'testChild',
  setup() {
    const test = ref(true);
    const testClick = () => {
      test.value = !test.value;
    };
    return { test, testClick };
  }
});
</script>
```
When i want use ElButton in the h render method, it not work.
But my self component was ok.
Is there anything wrong with what I wrote.

**Comment by npm-ued** [comment]
This is  error message
```
Uncaught (in promise) ReferenceError: ElButton is not defined
    at Proxy.render (test.vue?ed24:9)
    at renderComponentRoot (runtime-core.esm-bundler.js?5c40:705)
    at componentEffect (runtime-core.esm-bundler.js?5c40:4044)
    at reactiveEffect (reactivity.esm-bundler.js?a1e9:42)
    at effect (reactivity.esm-bundler.js?a1e9:17)
    at setupRenderEffect (runtime-core.esm-bundler.js?5c40:4027)
    at mountComponent (runtime-core.esm-bundler.js?5c40:3985)
    at processComponent (runtime-core.esm-bundler.js?5c40:3945)
    at patch (runtime-core.esm-bundler.js?5c40:3556)
    at componentEffect (runtime-core.esm-bundler.js?5c40:4126)
```

**Comment by BeADre** [comment]
it works well in my test, can you provide a repo or link of test?

```
import { defineComponent, h } from 'vue'
import { ElButton } from 'element-plus'

export default defineComponent({
  render() {
    return h(ElButton, {}, 'test')
  },
})
```

**Comment by npm-ued** [comment]
The repo [link](https://github.com/npm-ued/renderDemo)  @BeADre Thank you

**Comment by JeremyWuuuuu** [comment]
What version are you using it?

**Comment by npm-ued** [comment]
"element-plus": "1.0.1-beta.24"
"vue": "3.0.3"
@JeremyWuuuuu

**Comment by JeremyWuuuuu** [comment]
Please upgrade it to the latest version using `element-plus@latest`

**Comment by JeremyWuuuuu** [comment]
Note that `ElementPlus` has been developing with a relatively fast pace. So we would like you to use `@latest` tag to get the latest version.

**Comment by npm-ued** [comment]
Using the @latest also has the issue

**Comment by JeremyWuuuuu** [comment]
Check this out.
https://codesandbox.io/s/awesome-taussig-msuwd?file=/src/App.vue

**Comment by JeremyWuuuuu** [comment]
Everything seems legit, the only possible reason that I could guess is that your bundler has something to do with this. Could you provide a minimal reproducible demo repo to us for debugging?

**Comment by JeremyWuuuuu** [comment]
Replying on the website seems time consuming for the both of us, so please do provide a minimal reproducible repo for us to find out what is causing this issue.

**Comment by JeremyWuuuuu** [comment]
Oh, sorry I missed the link you provided. Sorry for that. I'll go investigate it.

**Comment by JeremyWuuuuu** [comment]
So I can confirm that this was cause by `babel-component-loader` because it import `element-plus/lib/el-button/index.js`, see the screenshot below. And because that this file was built in `cjs` format. 
![image](https://user-images.githubusercontent.com/15975785/106087494-18281700-615f-11eb-8f83-19ab0f309b88.png)

Because of that this is not a harmony export member A.K.A. it is not a ESModule, webpack goes to this:
![image](https://user-images.githubusercontent.com/15975785/106088169-6558b880-6160-11eb-8d84-3f9ba9355862.png)
then we resolve this code:
`h(getter, {}, []); // getter is what we got resolved by webpack.`

Here's how to solve that temporarily:
Assigning it to a variable, then the getter gets called, you get the actual component.
![image](https://user-images.githubusercontent.com/15975785/106088272-a18c1900-6160-11eb-9643-1dbcb8205542.png)
